furious/context/context.py

```diff
@@ -244,6 +249,10 @@ def add(self, target, args=None, kwargs=None, **options):
 
         target.update_options(_context_id=self.id)
 
+        if target.get_queue() == ASYNC_DEFAULT_QUEUE and (
+            self._default_queue != ASYNC_DEFAULT_QUEUE):
+            target.update_options(queue=self._default_queue)
```

Comment:
This seems like a good change, but the way it is implemented here will override the queue on Asyncs that were explicitly set to use default. Consider:
```python
with furious.context.new(queue='not-default'):
    ctx.add(fn, args=(some, args), queue='default')
    ctx.add(some_other_fn, args=(some, args))
```